RecoPixelVertexing/PixelTrackFitting/python/PixelTracks_cff.py

```diff
@@ -132,7 +129,7 @@
 
 # "Patatrack" sequence running on GPU (or CPU if not available)
 from Configuration.ProcessModifiers.gpu_cff import gpu
-(pixelNtupletFit & gpu).toModify(pixelTracksCPU,
+(pixelNtupletFit & gpu).toModify(pixelTracksSoA.cpu,
```

Comment:
The `pixelTracksCPU` appeared to be used only in this file, and this customization and the `run3_common.toModify(pixelTracksSoA.cpu,` on line 109/106 appeared to assume the `pixelTracksCPU` and `pixelTracksSoA.cpu` point to the same module object.